### DISCUSSION TITLE
More than 3 axes for line series plot

### DISCUSSION THREAD

**Discussion by sivarammurugan** [epezent/implot#297] [Q&A]
Hi , thanks to developers for this amazing library.  Currently implot supports upto 3 Y-axis to be specified for line plots. Is there any way to increase this limit ,say from 3 to 8 ?

**Comment by epezent** [comment]
In theory you should be able to do this by adding additional ImAxis enums (eg ImAxis_Y4,...ImAxisY8), but I have not tested this yet. Let me know what happens 😄
